tests/Browser/UserTest.php

```diff
@@ -8,62 +8,112 @@
 use App\User;
 class UserTest extends DuskTestCase
 {
+    /**
+     * Permisson account admin
+     *
+     * @return $admin
+     */
+    public function accountAdmin()
+    {
+        $admin = User::where('email', 'loitq@lifull-tech.vn')->first();
+
+        return $admin;
+    }
     /**
      * Admin logged go to dashboard
-     * Case 1-1
+     * case 1-1
+     * @return void
      */
     public function testLoginToPageAdmin()
     {
-        $admin = User::where('email', 'loitq@lifull-tech.vn')->first();
+        $admin = $this->accountAdmin();
 
         if (!$admin) {
             $admin = factory(User::class)->create([
                 'email' => 'loitq@lifull-tech.vn',
                 'password' => bcrypt('lifull@123'),
             ]);
         }
-        
+
         $this->browse(function (Browser $browser) use ($admin) {
             $browser->visit('/login')
-                    ->type('email', $admin->email)
-                    ->type('password', 'lifull@123')
-                    ->press('Login')
-                    ->assertPathIs('/admin');
+                ->type('email', $admin->email)
+                ->type('password', 'lifull@123')
+                ->press('Login')
+                ->assertPathIs('/admin');
         });
     }
 
     /**
      * Admin redirect to page create user
      * Case 1-2
+     * 
+     * @return void
      */
     public function testClickMenuToPageCreate()
     {
-        $admin = User::where('email', 'loitq@lifull-tech.vn')->first();
+        $admin = $this->accountAdmin();
         $this->browse(function (Browser $browser) use ($admin) {
             $browser->loginAs($admin)
-                ->visit('/admin')
-                ->visit(
-                    $browser->attribute('#link-create-user', 'href')
-                )
+                ->visit('/admin/user/list')
+                ->click('@createUser')
                 ->assertPathIs('/admin/user/create');
         });
     }
 
     /**
      * Admin redirect to page edit user
      * Case 1-3
+     * 
+     * @return void
      */
     public function testClickMenuToPageEdit()
     {
-        $admin = User::where('role', \App\User::ROLE_ADMIN);
-        $this->browse(function (Browser $browser) use ($admin){
-            $idUser = 2;
-            $browser
+        $admin = $this->accountAdmin();
+        $this->browse(function (Browser $browser) use ($admin) {
+            $browser->loginAs($admin)
+                ->visit('/admin/user/list')
+                ->click('@editUser2')
+                ->assertPathIs('/admin/user/edit/2');
+        });
+    }
+
+    /**
+     * Admin delete user
+     * case 2-15
+     *
+     * @return void
+     */
+    public function testPopupDeleteUserPressNoButton()
+    {
+        $admin = $this->accountAdmin();
+        $this->browse(function (Browser $browser) use ($admin) {
+            $browser->loginAs($admin)
+                ->visit('/admin/user/list')
```

Comment:
In your test file, '/admin/user/list' re-use multiple time. 
Why don't make constant for it and call this constant. 
Refer: 
https://github.com/loitq/training/blob/f8051435a4240b760a72b9369680d74c0f31585f/config/constants.php#L4-L7